Gopkg.toml

```diff
@@ -22,7 +22,7 @@
 
 [[constraint]]
   name = "github.com/cloudtrust/common-service"
-  branch = "master"
+  branch = "CLOUDTRUST-2109_authorizationsManagement"
   #version = "v1.2.1"
```

Comment:
Do you know what comment I have here ? :D

Reply:
I guess ;)
It will be fixed just before merge. It is needed to make it compile